lte/gateway/c/li_agent/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ PROJECT(LIAgent)
 
 include($ENV{MAGMA_ROOT}/orc8r/gateway/c/common/CMakeProtoMacros.txt)
 
-if (NOT BUILD_TESTS)
+if (BUILD_TESTS)
```

Comment:
I think we can get rid of this condition. I had to put this in on SessionD because one of the unit tests crashes with asan enabled. If these tests have an issue with asan, we can file an issue to fix it later though. If we need to keep the condition, it should be `NOT BUILD_TESTS`